back/iarbre_data/models.py

```diff
@@ -133,10 +134,77 @@ def save_mvt(self, mvt_data, filename):
         self.save()
 
     def __str__(self):
-        return f"Tile {self.model_type}/{self.zoom_level}/{self.tile_x}/{self.tile_y}"
+        return f"{self.model_type}/{self.zoom_level}/{self.tile_x}/{self.tile_y}"
 
 
 @receiver(pre_delete, sender=MVTTile)
 def before_delete_mvt_tile(sender, instance, **kwargs):
     """Delete the file when the model is deleted."""
     instance.mvt_file.delete(save=False)
+
+
+class Lcz(models.Model):
+    """Elementary element on the map with the value of the LCZ description."""
+
+    geometry = PolygonField(srid=2154)
+    map_geometry = PolygonField(srid=TARGET_MAP_PROJ, null=True, blank=True)
+    lcz_indice = models.CharField(max_length=4, null=True)
```

Comment:
```suggestion
    lcz_index = models.CharField(max_length=4, null=True)
```
si tu parles d'un indice, il me semble que c'est mieux traduit par `index`